doc/api/events.markdown

```diff
@@ -104,7 +104,7 @@ Note that `emitter.setMaxListeners(n)` still has precedence over
 
 ### emitter.listeners(event)
 
-Returns an array of listeners for the specified event.
+Returns a copy of an array of listeners for the specified event.
```

Comment:
That seems like an implementation detail that shouldn't be relevant to a user of the API.